contracts/pools/PoolCollection.sol

```diff
@@ -848,21 +891,15 @@ contract PoolCollection is IPoolCollection, Owned, BlockNumber, Utils {
     function _poolWithdrawalAmounts(
         Token pool,
         Pool memory data,
-        uint256 poolTokenAmount
+        uint256 poolTokenAmount,
+        uint256 reserveTokenAmount,
```

Comment:
Probably better to use the original variable name here, i.e., `baseTokensWithdrawalAmount` instead of `reserveTokenAmount`, or by the least, something which starts with 'baseToken', in order to avoid having "3 different types of tokens" appearing in this context (pool, base, reserve).

Reply:
Agreed 👍